test/e2e/info_test.go

```diff
@@ -49,4 +53,32 @@ var _ = Describe("Podman Info", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 		Expect(session.IsJSONOutputValid()).To(BeTrue())
 	})
+	It("podman info rootless storage path", func() {
+		if !rootless.IsRootless() {
+			Skip("test rootless_storage_path for rootless")
+		}
+		SkipIfRemote()
+		configPath := "/etc/containers/storage.conf"
+		desPath := "/etc/containers/storage.conf.test"
+		cmd := exec.Command("sudo", "cp", configPath, desPath)
+		out, err := cmd.CombinedOutput()
+		fmt.Println(string(out))
+		Expect(err).To(BeNil())
+
+		rootStoragePath := `"/tmp/$HOME/$USER/$UID"`
+		cmd = exec.Command("sudo", "sed", "-i", "-e", "/rootless_storage_path/d", "-e", fmt.Sprintf("/\\[storage\\]/a rootless_storage_path = %s", rootStoragePath), configPath)
```

Comment:
Again, if you're really rootless, you could just `cat` these two lines to the rootless storage.conf. (Even better: is there a command-line or envariable way to override the default `storage.conf` path? I couldn't find one)

Reply:
I can't find that envariable. I can try to add an variable for test and use $HOME/.config/containers without sudo.